scripts/build_database.sh

```diff
@@ -71,21 +71,19 @@ Dependencies:
   This script requires some non-standard dependencies to be installed before it can be used. This is a list of these
   items (which can normally be installed through your package manager):
 
-  * maven
-  * node-js
   * curl
   * pv
   * pigz
-  * java
   * uuidgen
   * parallel
+  * lz4
 END
 }
 
 # This function removes all temporary files that have been created by this script.
 clean() {
 	# Clean contents of temporary directory
-	rm -rf "$TEMP_DIR/$UNIPEPT_TEMP_CONSTANT"
+#	rm -rf "$TEMP_DIR/$UNIPEPT_TEMP_CONSTANT"
```

Comment:
I think this needs to be uncommented again? Or is there a reason why the temporary files are no longer deleted?